lib/PuppeteerSharp/Input/Mouse.cs

```diff
@@ -149,5 +149,114 @@ public Task WheelAsync(decimal deltaX, decimal deltaY)
+        /// </summary>
+        /// <param name="x">x coordinate</param>
+        /// <param name="y">y coordinate</param>
+        /// <param name="data">Drag data containing items and operations mask.</param>
+        /// <returns>A Task that resolves when the message was confirmed by the browser</returns>
+        public Task DragEnterAsync(decimal x, decimal y, DragData data)
+            => _client.SendAsync(
+                "Input.dispatchDragEvent",
+                new InputDispatchDragEventRequest
+                {
+                    Type = DragEventType.DragEnter,
+                    X = x,
+                    Y = y,
+                    Modifiers = _keyboard.Modifiers,
+                    Data = data,
+                });
+
+        /// <summary>
+        /// Dispatches a `dragover` event.
+        /// </summary>
+        /// <param name="x">x coordinate</param>
+        /// <param name="y">y coordinate</param>
+        /// <param name="data">Drag data containing items and operations mask.</param>
+        /// <returns>A Task that resolves when the message was confirmed by the browser</returns>
+        public Task DragOverAsync(decimal x, decimal y, DragData data)
+            => _client.SendAsync(
+                "Input.dispatchDragEvent",
+                new InputDispatchDragEventRequest
+                {
+                    Type = DragEventType.DragOver,
+                    X = x,
+                    Y = y,
+                    Modifiers = _keyboard.Modifiers,
+                    Data = data,
+                });
+
+        /// <summary>
+        /// Dispatches a `drop` event.
+        /// </summary>
+        /// <param name="x">x coordinate</param>
+        /// <param name="y">y coordinate</param>
+        /// <param name="data">Drag data containing items and operations mask.</param>
+        /// <returns>A Task that resolves when the message was confirmed by the browser</returns>
+        public Task DropAsync(decimal x, decimal y, DragData data)
+            => _client.SendAsync(
+                "Input.dispatchDragEvent",
+                new InputDispatchDragEventRequest
+                {
+                    Type = DragEventType.Drop,
+                    X = x,
+                    Y = y,
+                    Modifiers = _keyboard.Modifiers,
+                    Data = data,
+                });
+
+        /// <summary>
+        /// Performs a drag, dragenter, dragover, and drop in sequence.
+        /// </summary>
+        /// <param name="startX">Start X coordinate</param>
+        /// <param name="startY">Start Y coordinate</param>
+        /// <param name="endX">End X coordinate</param>
+        /// <param name="endY">End Y coordinate</param>
+        /// <param name="delay">If specified, is the time to wait between `dragover` and `drop` in milliseconds.</param>
+        /// <returns></returns>
+        /// <exception cref="System.NotImplementedException"></exception>
```

Comment:
Is this `NotImplementedException` tag still needed in the xml doc?

Reply:
No. Removed :)